packages/angular/build/src/builders/unit-test/schema.json

```diff
@@ -65,6 +65,11 @@
       "items": {
         "type": "string"
       }
+    },
+    "providersFile": {
+      "type": "string",
+      "description": "TypeScript file that exports an array of Angular providers to use during test execution. The array must be a default export.",
+      "minLength": 1
```

Comment:
is `"minLength": 1` intentional?

Reply:
yes. attempts to prevent an empty string as a value.